### PR TITLE
[WIP] First stab at weather vane documentation

### PR DESCRIPTION
**Pull request by RomanBapst**
@hamishwillee This is documentation for the weathervane feature which we will merge soonish.
https://github.com/PX4/Firmware/pull/10411

Btw, my next task is to bring in the optical flow driver for the bitcraze flow deck so that goes along with
https://github.com/PX4/px4_user_guide/pull/304
:-)

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Table of contents
+# Summary
```
Comment (hamishwillee) [inline SUMMARY.md]: FYI, this makes no difference

**Comment by hamishwillee** [comment]
@RomanBapst Looks great. I have subedited to my taste and cross linked to related docs :-) 

The main things for you to remember for future docs are that in user land the names for [flight modes](https://docs.px4.io/en/getting_started/flight_modes.html) are what the user sees in QGC not state machine names. So Mission Mode, not Auto Mode (there are a bunch of Auto modes). 

OK to just push this in when the firmware PR goes in?

Good luck with 304 :-)

**Comment by RomanBapst** [comment]
@hamishwillee Thanks a lot! I fixed a small typo and merged as the PR on the firmware side was also just merged.

**Comment by hamishwillee** [comment]
There goes my reputation for infallibility :-(. Thanks! We'll have to update this to say that is is default-on after you make that change.

**Comment by RomanBapst** [comment]
@hamishwillee Well, what should I say now that I notice that my commit message to fix the typo has a typo.

**Pull request merged**